gui_qt/work_modes.py

```
@@ -235,6 +237,25 @@ class WorkModeSpec:
     ),
+    WorkMode.FINAL_REVIEW: WorkModeSpec(
+        mode=WorkMode.FINAL_REVIEW,
+        category=TaskCategory.MAINTENANCE,
+        label="最终审校",
+        start_button_label="开始最终审校",
+        resume_button_label="继续审查",
+        task_group_label="最终审校任务",
+        progress_tab_label="审查进度",
+        writeback_tab_label="订正候选",
+        idle_workflow_heading="尚未开始最终审校",
+        idle_workflow_message="先生成问题报告；只有人工选择的问题才会进入订正预览，绝不直接修改脚本。",
```

Ruff (0.15.21)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)